crane-cloud/frontend
# chore: refactor manage-sections for [project/app/database] on user/admin sides

# Description

- align the manage-sections for [projects/apps/databases] on both the user and admin side
- remove duplicate code and use reusable component for consistency

## Type of change


## Trello Ticket ID

https://trello.com/c/4xPGjJuW

## How Can This Been Tested?

- Run this branch locally and checkout all the [project/app/databases]-manage sections under their settings pages to notice the changes.

## Checklist:


# Screenshots

<img width="1009" alt="Screenshot 2023-12-05 at 22 37 29" src="https://github.com/crane-cloud/frontend/assets/63339234/7673ccc0-29ea-40c6-8a42-dab7ccef5a74">

<img width="984" alt="Screenshot 2023-12-06 at 00 04 50" src="https://github.com/crane-cloud/frontend/assets/63339234/cd7106c6-b1a1-4c0f-8e73-095f21d7aee2">
